Require error after upgrade to 0.1.15

After upgrading to latest version 0.1.15 from some earlier version (not sure which) I get the warning below, which results in the node not loading in the Node-RED UI.

```
node-red-dev    | 3 Sep 07:50:34 - [warn] [node-red-contrib-yr/weather-info] Error: Cannot find module 'haversine-distance'
node-red-dev    | Require stack:
node-red-dev    | - /data/node_modules/node-red-contrib-yr/yr.js
node-red-dev    | - /data/node_modules/node-red-contrib-yr/weather-info/weather-info.js
node-red-dev    | - /usr/src/node-red/node_modules/@node-red/registry/lib/loader.js
node-red-dev    | - /usr/src/node-red/node_modules/@node-red/registry/lib/index.js
node-red-dev    | - /usr/src/node-red/node_modules/@node-red/runtime/lib/nodes/index.js
node-red-dev    | - /usr/src/node-red/node_modules/@node-red/runtime/lib/index.js
node-red-dev    | - /usr/src/node-red/node_modules/node-red/lib/red.js
node-red-dev    | - /usr/src/node-red/node_modules/node-red/red.js
```

Uninstalling > Re-installing the module solved the problem.

--- a/yr/package.json
+++ b/yr/package.json
@@ -21,7 +21,7 @@
         "url" : "http://plus.google.com/+JoTorsmyr"
     },
     "dependencies": {
-        "node-fetch": "~2.6.x",
+        "node-fetch": "~2.7.0",
         "haversine-distance": "^1.2.1",
         "webidl-conversions": "^7.0.0"
     }

--- a/yr/package.json
+++ b/yr/package.json
@@ -21,6 +21,7 @@
         "url" : "http://plus.google.com/+JoTorsmyr"
     },
     "dependencies": {
-        "node-fetch": "~2.6.x"
+        "node-fetch": "~2.6.x",
+        "haversine-distance": "^1.2.1"
     }
 }

--- a/yr/package.json
+++ b/yr/package.json
@@ -22,6 +22,7 @@
     },
     "dependencies": {
         "node-fetch": "~2.6.x",
-        "haversine-distance": "^1.2.1"
+        "haversine-distance": "^1.2.1",
+        "webidl-conversions": "^7.0.0"
     }
 }

--- a/yr/package.json
+++ b/yr/package.json
@@ -5,7 +5,7 @@
     "repository" : {
         "type" :"git",
         "url" : "https://github.com/tverilytt/NODE-Red/tree/master/yr"
-    },    
+    },
     "license" : "Apache",
     "keywords" : [ "node-red", "weather-data", "weather-info" ],
     "node-red" : {

